lifecycle-operator/controllers/lifecycle/schedulinggates/controller_test.go

```diff
@@ -264,6 +264,42 @@ func TestSchedulingGatesReconciler_Reconcile(t *testing.T) {
 			wantErr:            false,
 			expectGatesRemoved: false,
 		},
+		{
```

Comment:
This test case is duplicate of the existing marked as "related WorkloadVersion is completed". I don't think we need this

Reply:
Alright I will remove it